src/qibosoq/rfsoc_server.py

```diff
@@ -11,18 +11,30 @@
 import socket
 import traceback
 from socketserver import BaseRequestHandler, TCPServer
+from typing import Dict, List
 
 from qick import QickSoc
 
 import qibosoq.configuration as cfg
-from qibosoq.components import Config, OperationCode, Pulse, Qubit, Sweeper
+from qibosoq.components.base import Config, OperationCode, Qubit, Sweeper
+from qibosoq.components.pulses import Pulse, pulse_class_from_shape
 from qibosoq.programs.pulse_sequence import ExecutePulseSequence
 from qibosoq.programs.sweepers import ExecuteSweeps
 
 logger = logging.getLogger(cfg.MAIN_LOGGER_NAME)
 qick_logger = logging.getLogger(cfg.PROGRAM_LOGGER_NAME)
 
 
+def from_dict_to_pulses(list_sequence: List[Dict]) -> List[Pulse]:
```

Comment:
```suggestion
def load_pulses(list_sequence: List[Dict]) -> List[Pulse]:
```
The exact type is in the hint, you can keep the function name simpler and more generic.